sections/data_set_files.md

```diff
@@ -1,6 +1,22 @@
 # DataSet Files
 
-**NOTE: These requests require the *DataAccess* API scope. (see [Scopes](scopes.md))**
+**NOTE: These requests require the *DataAccess* API scope (see [Scopes](scopes.md)) AND a request for processing the specified Study Programs to the data sets before each query.**
```

Comment:
Wording is not passive voice, but is still a little confusing.  Consider rewording?

"AND a request for processing the specified Study Programs to the data sets before each query."